common.lic

```diff
@@ -95,13 +95,23 @@ module DRC
   # before performing command and do smart retries.
   # Will wait for matching text up to 15 seconds then timeout.
   def bput(message, *matches)
-    wput(message, 15, matches)
+    options = (matches.shift if matches.first.is_a?(Hash)) || {}
+    wput(message, options, matches)
   end
 
   # Like `fput` but better because will wait for RT
   # before performing command and do smart retries.
   # Will wait for matching text up til the given timeout in seconds.
-  def wput(message, timeout, *matches)
+  def wput(message, options, *matches)
+    timeout = (options['timeout'] ||= 15)
```

Comment:
Should 15 be a property of `wput` or `bput`?

Reply:
Yeah, that makes more sense that it's part of `bput` so that `wput` is simpler, just running with whatever is given to it. I've made that change per https://github.com/rpherbig/dr-scripts/pull/4945/commits/bfa5a0d2fd436b6d5cca8c136a4b312b99dcac32

With this change, technicaly, we don't need `wput` method anymore (which I originally introduced so that I had a way to specify custom timeouts for scripts like `feed-cloak`).

Do you think we should keep `wput` or consolidate back down to just `bput`?

Reply:
I think just keeping bput is fine, but I don't have a strong preference

Reply:
I'll combine them again. The only use of `wput` I found was in `feed-cloak` which is the script for which I introduced `wput` for